requirements.txt

```diff
@@ -54,6 +54,7 @@ pytz==2018.4
 antlr4-python2-runtime==4.7.1
 statsd==3.2.1
 retrying==1.3.3
+PyAthena[SQLAlchemy]
```

Comment:
hey, where do you use the dependency?

Reply:
From what I understood it's the extension of SQLAlchemy Athena